.evergreen/evergreen.yml.in

```diff
@@ -873,34 +873,54 @@ functions:
   papertrail_trace:
     - command: shell.exec
       params:
+        env:
+          PAPERTRAIL_KEY_ID: ${papertrail_key_id}
+          PAPERTRAIL_SECRET_KEY: ${papertrail_secret_key}
+          EVERGREEN_TASK_ID: ${task_id}
+          EVERGREEN_EXECUTION: ${execution}
+          EVERGREEN_AUTHOR: ${author}
         working_dir: src
         shell: bash
         script: |
           set -e
-          set -x
```

Comment:
Can we write the secrets first, then turn `set -x` on after so that it's turned on for the rest of the script? It's really helpful when debugging CI issues